src/Context/Auth/Auth.js

```diff
@@ -0,0 +1,70 @@
+import { createContext } from "react";
+import { useContext } from "react";
+import axios from "axios";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+const AuthContext = createContext();
+const useAuth = () => useContext(AuthContext);
+
+const AuthProvider = ({children})  => {
+  const [loader,setLoader] = useState(false)
+  const navigate = useNavigate()
+
+
+ /* login data */ 
+
+const encodedToken = localStorage.getItem("token");
+const [login, setLogin] = useState([]);
+const fetchLoginDetails = async (email,password) => {
```

Comment:
You can put these helper methods in a different file and import them here, It will make your code look clean.

Reply:
Thank you . Have changed for now.